{
  "Header": {
    "Original": "CVE-2020-29396 (A sandboxing issue in Odoo Community 11.0 through 13.0 and Odoo Enterp ...)",
    "Line": 187655,
    "ID": "CVE-2020-29396",
    "Description": "(A sandboxing issue in Odoo Community 11.0 through 13.0 and Odoo Enterp ...)"
  },
  "Annotations": [
    {
      "Original": "- odoo \u003cnot-affected\u003e (Fixed before initial upload to Debian)",
      "Line": 187656,
      "Type": "package",
      "Package": "odoo",
      "Kind": "not-affected",
      "Description": "Fixed before initial upload to Debian"
    },
    {
      "Original": "NOTE: https://github.com/odoo/odoo/issues/63712",
      "Line": 187657,
      "Type": "NOTE",
      "Description": "https://github.com/odoo/odoo/issues/63712"
    }
  ]
}